ballerina/build.gradle

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2024 WSO2 LLC. (https: *www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+buildscript {
+    repositories {
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/plugin-gradle'
+            credentials {
+                username System.getenv('packageUser')
+                password System.getenv('packagePAT')
+            }
+        }
+    }
+    dependencies {
+        classpath "io.ballerina:plugin-gradle:${project.ballerinaGradlePluginVersion}"
+    }
+}
```

Comment:
Shall we move the dependency management to the `settings.gradle` file? Check other repos like [gmail](https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/e211d4e622c4b4a43f5d47a4b0e06e74297ca4b1/settings.gradle#L14).